src/core-manager/index.js

```diff
@@ -304,12 +304,29 @@ export class CoreManager extends TypedEmitter {
       peer._maybeWant(0, core.length)
     })
 
-    // A non-writer core will emit 'append' when its length is updated from the
-    // initial sync with a peer, and we will not have sent a "maybe want" for
-    // this range, so we need to do it now. Subsequent appends are propogated
-    // (more efficiently) via range broadcasts, so we only need to listen to the
-    // first append.
```

Comment:
I moved this comment down and re-formatted it so no line was longer than 80 characters; I didn't change it other than fixing a typo.